### ISSUE TITLE
Does FSDP works with DDP ?

### ISSUE THREAD

**Issue by Musa-Sina-Ertugrul** [pytorch/pytorch#157917]
I run 2 processes if I pass model to _FSDP_ before DDP, Does it work ?

**Comment by weifengpy** [comment]
good question! see full example below. roughly it is 
```
fully_shard(model, ignored_params=params_for_ddp)
DDP._set_params_and_buffers_to_ignore_for_model(
    module=model,
    params_and_buffers_to_ignore=params_for_fsdp,
)
model = DDP(model, broadcast_buffers=False)
```

https://github.com/pytorch/pytorch/blob/c54778625ec45b94eb97f52da3ad6c2eb3d3c3c9/test/distributed/_composable/fsdp/test_fully_shard_ignore_params.py#L283-L287

**Comment by Musa-Sina-Ertugrul** [comment]
Thanks <3 , But Can you clarify my understanding, ( Background: I have two gpu and currently I can pretrain smal lm Therefore I want using FSDP with DDP for changing model parameters and gradient [ I have enough bus speed ] interchangely ) with your suggested example I can not do that I can only do change parameters have been locked for grads ? Am I correct ? If not Can you explain

**Comment by aditvenk** [comment]
Wei's response shows the supported composition -- statically split parameters between FSDP & DDP. One ignores the params intended for the other. Ignored does not mean frozen; those params can still require grads and be updated. The linked test shows this.

**Comment by aditvenk** [comment]
Pl re-open issue or file a new one if you have other questions. Thank you!
